deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml

```diff
@@ -1,9 +1,9 @@
 {{- $yamlFile := toYaml $.Values.otelcol.config }}
 {{- $_collector := .Values.sumologic.collectorName | default .Values.sumologic.clusterName | quote }}
-{{- $sourceName := .Values.sumologic.sourceName | quote }}
-{{- $sourceCategory := .Values.sumologic.sourceCategory | quote }}
-{{- $sourceCategoryPrefix := .Values.sumologic.sourceCategoryPrefix | quote }}
-{{- $sourceCategoryReplaceDash := .Values.sumologic.sourceCategoryReplaceDash | quote }}
+{{- $sourceName := .Values.fluentd.logs.containers.sourceName | quote }}
+{{- $sourceCategory := .Values.fluentd.logs.containers.sourceCategory | quote }}
+{{- $sourceCategoryPrefix := .Values.fluentd.logs.containers.sourceCategoryPrefix | quote }}
+{{- $sourceCategoryReplaceDash := .Values.fluentd.logs.containers.sourceCategoryReplaceDash | quote }}
```

Comment:
I see it's sprinkled all over this file but why would the traces config be based on `.Values.fluentd.logs.containers.*` configuration?

Reply:
AFAIK, To not duplicate configuration for source headers

Reply:
seems to be pretty old, probably from times when there was only one place to configure source headers for logs

Reply:
@pmalek-sumo I've changed it to fit this document https://github.com/SumoLogic/sumologic-kubernetes-collection/blame/release-v1.3/deploy/docs/v1_migration_doc.md#L50-L53

Reply:
This looks good to me. From the doc mentioned above by @pmatyjasek-sumo , the `sumologic.source*` properties haven't been available in the chart since v1.0.0. It's about time tracing configuration is updated. 😄

Reply:
Please create issue next time if there is something to do ;)

Reply:
Ok @sumo-drosiek :)

Reply:
@pmatyjasek-sumo @astencel-sumo I mean this: #1583. Issue without PR is fine, but lets track future (especially breaking) changes which are result of discussions :)